src/pages/ProfilePage/components/UserPosts/UserPosts.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+import styles from "./Posts.module.scss";
+
+const Posts: FC = () => {
+  return (
+    <div className={styles["posts-wrapper"]}>
+      {[...new Array(4)].map((post, index) => (
```

Comment:
An error happens with "post" param too. Use an underscore symbol to pass the building process